README.rst

```diff
@@ -7,7 +7,7 @@ Real Time Locator System
         :target: https://pypi.python.org/pypi/rtls
 
 .. image:: https://img.shields.io/travis/release-depot/rtls.svg
-        :target: https://travis-ci.org/release-depot/rtls
+        :target: https://travis-ci.com/release-depot/rtls
```

Comment:
Is the switch to '.com' required? I know it is available, but does '.org' now work anymore?

Reply:
The cookie cutter update did that, but I can revert that bit back